src/index.html

```diff
@@ -5,9 +5,48 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+      >
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="card card--width" data-qa="card">
+      <img
+        class="card__photo"
+        src="./images/imac.jpeg"
+        alt="Imac Monoblock photo"
+      >
+      <h1 class="card__header">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__rating">
+        <div class="stars stars__star--0">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+         Reviews: 5
+      </div>
+      <p class="card__price">
+        <span class="card__title">Price:</span>
+        <span class="card__sum">$2,199</span>
+      </p>
```

Comment:
`.card__title` is definitely not 'title'. I suggest `.card__price-text` and `.price__price-value`.
Good job on using `<p>` and `<span>`🔥